src/signals/incident-management/containers/IncidentDetail/components/Detail/components/ExtraProperties/index.js

```diff
@@ -19,14 +19,18 @@ const getValue = answer => {
   return answer;
 };
 
-const ExtraProperties = ({ items }) =>
-  items &&
-  items.map(item => (
+const ExtraProperties = ({ items }) => {
+  // Some incidents have been stored with values for their extra properties that is incompatible with the current API
+  // We therefore need to check if we're getting an array or an object
+  const itemList = Array.isArray(items) ? items : Object.entries(items);
```

Comment:
I would mention here that the `good` last version of the interface is the array. It is said in the PR description but later on will be forgotten.

Reply:
The prop types already mention that the type of `items` needs to be an array, so no need to be more explicit than that.